TreningGenerator/_AbstractTypes/ExerciseProtocol.swift

```diff
@@ -1,7 +1,13 @@
 
 import Foundation
 
+enum ExerciseKind: String {
```

Comment:
You can add helper methods to this enum eg. `var allCases: [ExerciseKind] = { return [. calisthenics, . streching] }`

Think where you could use this method/property :)

Reply:
fixed